api/swagger.yaml

```diff
@@ -1837,6 +1844,13 @@ definitions:
         description: "The ID of the swarm."
         type: "string"
       Version:
+        description: |
+          The version number of the swarm object being. This is required to avoid conflicting writes.
```

Comment:
oh, interesting, yes, I see the PR is updated. Odd that GitHub didn't complain, and still allowed me to comment